src/todo_final.ts

```diff
@@ -0,0 +1,88 @@
+//import type { Config } from "@jest/types";
+//this array will contain all the todo list items
+let todoItems: any[] = [];
```

Comment:
This array can be typed as an array of TodoItems: 
```
type TodoItem = {
	text: string,
        ...other key types go here
}
let todoItems: TodoItem[] = [];
```

This way TypeScript will tell you if you try to do something incorrect with an element of `todoItems`.